x-pack/plugins/grokdebugger/public/sections/grokdebugger/components/apply_editor_options.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EDITOR } from '../../../../common/constants';
+
+export const applyEditorOptions = (editor, minLines, maxLines) => {
```

Comment:
Not a big deal but if you `export function applyEditorOptions(editor, minLines, maxLines) { ... }` here instead, the function name will show up in stack traces, which can be helpful when debugging.

Reply:
👍